lightautoml/dataset/roles.py

```diff
@@ -130,7 +131,16 @@ def from_string(name: str, **kwargs: Any) -> "ColumnRole":
 
 
 class NumericRole(ColumnRole):
-    """Numeric role."""
+    """Numeric role.
```

Comment:
Use the text from the removed docstring "Create numeric role with specific numeric dtype."